flatpak-builder/index.js

```diff
@@ -245,7 +240,9 @@ const build = async (manifest, manifestPath, cacheHitKey, config) => {
 
   if (config.cacheBuildDir && (cacheKey !== cacheHitKey)) {
     await cache.saveCache(
-      CACHE_PATH,
+      [
+        '.flatpak-builder'
+      ],
```

Comment:
```suggestion
      [...CACHE_PATH], // TODO: drop once https://github.com/actions/toolkit/pull/1378 is merged
```
is better for me 

also you will have to rebuild the index.js, see contributing guidelines

Reply:
@bilelmoussaoui 
I made the modifications as required. However, when I compiled using ncc, the generated files differed significantly from the previous ones. I checked the previous pull requests, but didn't find this issue. Since I'm not very familiar with node.js and related things, I manually edited the dist/index.js.
I hope it works fine now.

Reply:
No, please just run ncc & push your changes. The file should never be modified manually

Reply:
Sorry, I know what to do now. Please wait a moment